src/rpcnet.cpp

```diff
@@ -165,6 +173,18 @@ UniValue getpeerinfo(const UniValue& params, bool fHelp)
         }
         obj.push_back(Pair("whitelisted", stats.fWhitelisted));
 
+        UniValue sendPerCmd(UniValue::VOBJ);
+        BOOST_FOREACH( const mapCmdSize::value_type &i, stats.mapSendBytesPerCmd) {
+            sendPerCmd.push_back(Pair(i.first, i.second));
+        }
+        obj.push_back(Pair("bytessent_per_cmd", sendPerCmd));
+
+        UniValue recvPerCmd(UniValue::VOBJ);
+        BOOST_FOREACH( const mapCmdSize::value_type &i, stats.mapRecvBytesPerCmd) {
+            recvPerCmd.push_back(Pair(i.first, i.second));
```

Comment:
I though if someone writes a statistics fronted that uses the RPC as data-source it would complicate things (a little bit) because you don't have a full list of available commands at the beginning. But no strong opinion. I think the drawback right now is a relatively big response.